File: src/main/java/com/kylecorry/frc/vision/detection/TargetGroupSpecs.java
package com.kylecorry.frc.vision.detection;

public interface TargetGroupSpecs {

	public static enum Alignment {
		LEFT, TOP
	}

	double getTargetWidthRatio();

	double getTargetHeightRatio();

	Alignment getAlignment();

	double getGroupWidth();

	double getGroupHeight();
}

File: src/main/java/com/kylecorry/frc/vision/detection/TargetGroupSpecs.kt
package com.kylecorry.frc.vision.detection

interface TargetGroupSpecs {

    enum class Alignment {
        LEFT, TOP
    }

    val targetWidthRatio: Double

    val targetHeightRatio: Double

    val alignment: Alignment

    val groupWidth: Double

    val groupHeight: Double
}
